database/migration/src/main/resources/changesets/call_caching_aggregation/call_caching_base_aggregation.sql

```diff
@@ -0,0 +1,18 @@
+INSERT INTO CALL_CACHING_AGGREGATION_ENTRY(
+  CALL_CACHING_ENTRY_ID,
+  BASE_AGGREGATION
+)
+  SELECT cche.CALL_CACHING_ENTRY_ID,
+  UPPER(MD5(GROUP_CONCAT(
+                cche.HASH_VALUE ORDER BY cche.HASH_KEY ASC SEPARATOR ''
```

Comment:
Imagine I have two `Array[File]` inputs, `a` and `b`. Without keys I think you would cache match the following two input sets:
```
a = [ file_1, file_2 ]
b = [ file_3, file_4 ]
```
and:
```
a = [ file_1 ]
b = [ file_2, file_3, file_4 ]
```